apps/projects/app/assets/svg/github.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="60" height="58">
```

Comment:
The icon is now in Shared/assets/components/GitHub.js (added, because it was needed for panels..)

Reply:
Solved int 402c5fbae1dbb3202e921f823f42741befc70c94